devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/tasks/QuarkusUpdate.java

```diff
@@ -33,13 +31,13 @@ public abstract class QuarkusUpdate extends QuarkusPlatformTask {
 
     @Input
     @Optional
-    public Boolean getNoRewrite() {
-        return noRewrite;
+    public Boolean getRewrite() {
+        return rewrite;
     }
 
-    @Option(description = "Disable the rewrite feature.", option = "noRewrite")
-    public QuarkusUpdate setNoRewrite(Boolean noRewrite) {
-        this.noRewrite = noRewrite;
+    @Option(description = "Run the suggested update recipe for this project using OpenRewrite.", option = "rewrite")
```

Comment:
this shuold be negatable - then you keep backwards compatible option.
And yes, no need imo to refer to OpenRewrite as there can be more than just openrewrite in play.

```suggestion
    @Option(description = "Run the suggested update recipe for this project.", option = "rewrite")
```